src/profileReport.js

```diff
@@ -182,12 +179,12 @@ ProfileReport.prototype.collectUnlisten = function(data, path) {
   this.collectSpeed(data, path, this.state.unlistenSpeed);
 };
 
-ProfileReport.prototype.collectConnect = function(data, path) {
-  this.collectSpeed(data, path, this.state.connectSpeed);
+ProfileReport.prototype.collectConnect = function(data) {
+  this.collectSpeed(data, "null", this.state.connectSpeed);
```

Comment:
do we still need this?

Reply:
We don't but changing it would mean creating another `collectSpeed` clone that really just hides the fact that we're using a sentinel path for aggregation. My preference would be to keep this, but I can make the clone if you prefer that.

Reply:
Let's find a way to do this without imposing special meaning on a specific (and valid!) sentinel path.